offline.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * The template for displaying offline pages
+ *
+ * @link https://github.com/xwp/pwa-wp#offline--500-error-handling
+ *
+ * @package wp_rig
+ */
+
+namespace WP_Rig\WP_Rig;
+
+// Prevent showing nav menus.
+add_filter( 'has_nav_menu', '__return_false' );
+
+get_header(); ?>
+
+	<main id="primary" class="site-main">
+
+		<section class="error-offline not-found">
+			<header class="page-header">
+				<h1 class="page-title"><?php esc_html_e( 'Oops! It looks like you&#8217;re offline.', 'wp-rig' ); ?></h1>
+			</header><!-- .page-header -->
+
+			<div class="page-content">
+				<p><?php esc_html_e( 'The page could not be loaded because you&#8217;re offline. Please try again once reconnected.', 'wp-rig' ); ?></p>
```

Comment:
I believe this will be redundant. This message will be added by the plugin for the placeholder.

See https://github.com/xwp/twentyseventeen-westonson/blob/03934b7328f9b18dd979ae97458ad38feb6f0e1a/offline.php#L27-L33